grails-app/controllers/au/org/ala/biocache/hubs/OccurrenceController.groovy

```diff
@@ -639,4 +638,19 @@ class OccurrenceController {
         data.count = qualityService.getExcludeCount(params.categoryLabel, profile.getCategories(), requestParams)
         render data as JSON
     }
+
+    def getAlerts() {
+        String userId = authService?.getUserId()
+        render ((userId == null) ? [error: 'userId must be supplied to get alerts'] : webServicesService.getAlerts(userId))  as JSON
```

Comment:
Error responses should have a non 200 series HTTP status code.

These error maps aren't checked for in the JS that calls them

Reply:
Solved. 

In JS the checkbox is initially hidden now. So only when call is successful will show it.